.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Release checklist
+about: Použi tento checklist na testovanie artefaktov pred public releaseom
+title: "Release "
+labels: release
+assignees: @slovensko-digital/autogram-release-team
+---
+
+## Windows
+
+- [ ] funguje inštalácia na Windows cez stiahnutý .msi
+- [ ] funguje spustenie v GUI móde
+- [ ] funguje URL handler [autogram://go](autogram://go)
+- [ ] funguje GUI otvoriť jeden súbor, ten sa zobrazí, viem ho podpísať, vytvorí sa podpísaný súbor
+
```

Comment:
Pridal by som ešte test, či viem otvoriť swagger a podpísať v ňom jeden example. Alebo si myslíš, že to je zbytočné? Ja si predstavujem, že tým zistíme, či sme nepikazili nejaké špecifické nastavenie servera.

A druhá vec: pridal by som CLI `autogram --help`, aby sme zistili, či sme nepokazili niečo s inštaláciou.